syntax = "proto3";
package common;


message GeoPoint {
double lat = 1;
double lon = 2;
double alt = 3;
}